Library/Homebrew/livecheck/strategy/hackage.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Homebrew
+  module Livecheck
+    module Strategy
+      module Hackage
+        module_function
+
+        def match?(url)
+          /hackage\.haskell\.org/.match?(url)
+        end
+
+        def find_versions(url, regex = nil)
+          package_name = ((url.split("/")[4]).split("-")[0..-2]).join("-")
```

Comment:
This would probably benefit from some example URLs in a comment (and in other strategies).

Reply:
I agree and I've been incorporating this sort of information in the aforementioned work I'm doing.

Reply:
👍🏻